localization/languages/es.json

```diff
@@ -54,7 +54,7 @@
     "searchHistory": "Historial de búsqueda",  //noun?
     "importBookmarks": "Importar marcadores",
     "exportBookmarks": "Exportar marcadores",
-    "runUserscript": "Correr userscript", //userscript doesn't have a Spanish translation, afaik
+    "runUserscript": "Correr userscript", //userscript doesn't have a Spanish translation, afaik maybe "script de usuario"
```

Comment:
Whatever translation you think is best here is fine with me.

Reply:
i think I'd leave it as userscript because if someone wants to know what it is it's easier to find it that way.

Reply:
OK, sounds good.